web/plugs/require_super_admin.ex

```diff
@@ -0,0 +1,25 @@
+defmodule OxleasAdhd.Plugs.RequireSuperAdmin do
+  import Plug.Conn
+
+  def init(opts), do: opts
+  def call(conn, _) do
+    current_user = conn.assigns[:current_user]
+    if current_user do
+      case Map.get(current_user, :role) do
+        "super_admin" ->
+          conn
+        _ ->
+          conn |> redirect_to_login
+      end
+    else
+      conn |> redirect_to_login
+    end
+  end
+
+  defp redirect_to_login(conn) do
```

Comment:
Maybe rename redirect to home? I think it is better to keep home as the route to redirect to, because it's possible a logged in user may try to go to those routes, and it's confusing to be redirected to login when you are already logged in.